config_cam_baseline_example.yaml

```
@@ -64,6 +64,10 @@ user: 'USER-NAME-NOT-SET'
 #This first set of variables specify basic info used by all diagnostic runs:
 diag_basic_info:
 
+    #History file string to match (eg. cam.h0 or ocn.pop.h.ecosys.nday1)
+    #Default: cam.h0. Only affects timeseries as everything else uses timeseries 
```

Should we add a warning here that there shouldn't be a trailing `.` at the end of the string?  It looks like the file globbing step includes the `.`, and so if someone has it in the string itself the file globbing will likely fail.  We also probably want to remove it from the example in the comment as well:

 ```suggestion
     #Default: cam.h0 Only affects timeseries as everything else uses timeseries 
```

---

Adding the comment and removing the '.' from the example. It seems we should really test if it ends with a '.' and if so, not add one, but is it okay to do that later?